src/styles/components/_footer.scss

```diff
@@ -0,0 +1,12 @@
+.footer {
+  height: 180px;
```

Comment:
at desktop version your footer too small,must be not less 359px,also footer must be sticky.
![image](https://github.com/mate-academy/layout_landing-page/assets/143347271/49572698-1ff0-4b90-bbc2-5d7e145b57ff)

Reply:
Improved that as well